plugin/src/main/java/com/denizenscript/denizen/objects/WorldTag.java

```diff
@@ -614,8 +615,10 @@ public static void register() {
             // @returns DurationTag
             // @description
             // Returns the relative in-game time of this world as a duration.
```

Comment:
replace the description with a deprecation notice as well. Look at other deprecated tags for example